spec/models/group_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'rails_helper'
+
```

Comment:
- [ ] Most features and components in your application are not tested. Kindly ensure to add integration test for your application, including requests, views and units test for other components of your app

Reply:
@rloterh , Have added more tests as requested

Reply:
Nice job!❤️
Kindly resubmit your project for another review
Cheers!